src/core_ocean/Registry.xml

```diff
@@ -444,7 +350,7 @@
 		/>
 	</nml_record>
 	<nml_record name="cvmix" mode="forward">
-		<nml_option name="config_use_cvmix" type="logical" default_value=".false." units="NA"
+		<nml_option name="config_use_cvmix" type="logical" default_value=".true." units="NA"
```

Comment:
@mark-petersen note that turning this on will not actually enable any mixing schemes.  I don't know if this is the intention for test cases.  To get PP by default you also need use ```config_use_cvmix_shear = .true.```

Reply:
I actually think the test cases are consistent with this setting.